scripts/vim-patch.sh

```diff
@@ -376,8 +376,16 @@ submit_pr() {
   pr_title="$(printf 'vim-patch:%s' "${pr_title#,}")"
 
   if [[ $push_first -ne 0 ]]; then
-    echo "Pushing to 'origin/${checked_out_branch}'."
-    output="$(git push origin "${checked_out_branch}" 2>&1)" &&
+    local push_remote
+    push_remote="$(git config --get branch."${checked_out_branch}".pushRemote)"
+    if [[ -z "$push_remote" ]]; then
+      push_remote="$(git config --get remote.pushDefault)"
+      if [[ -z "$push_remote" ]]; then
+        push_remote="origin"
```

Comment:
Probably still makes sense to use `git config --get branch.<name>.remote` here.

---
Alternatively, would it be better to do something like `find_get_remote` but for the user's repo?

```sh
find_push_remote() {
  git_remote=$(git remote -v \
    | awk '$2 !~ /github.com[:\/]neovim\/neovim/ && $3 == "(push)" {print $1; exit}')
  if [[ -z "$git_remote" ]]; then
    git_remote="origin"
  fi
  echo "$git_remote"
}
```

Reply:
Maybe `branch.<name>.remote` can used if it is not Neovim upstream remote.